adb_graphics/default_specs.yml

```diff
@@ -740,6 +740,56 @@ icmr: # Ice Water Mixing Ratio
         - ICMR_P0_L105_{grid}
         - CIMIXR_P0_L105_{grid}
       prs: ICMR_P0_L100_{grid}
+icprb: # Icing Probability
+  500ft: &icprb
+    clevs: !!python/object/apply:numpy.arange [5, 86, 10]
+    cmap: gist_ncar
+    colors: icprb_colors
+    ncl_name: ICPRB_P0_L102_{grid}
+    ticks: 10
+    transform: conversions.percent
+    unit: "%"
+    vertical_index: 0   # grib2 vertical levels are in meters,
+                        # so using index here to avoid errors from conversions
```

Comment:
Would you mind leaving an additional comment in the code about how you chose these particular index values? Like "looked at an example file, and eyeballed which level was close to the right level" might even be enough of a clue as to how to do this in the future if something changes.

Just some kind of hint as to how these numbers were derived.